CefSharp/WebBrowserExtensions.cs

```diff
@@ -309,7 +309,7 @@ public static bool LoadHtml(this IWebBrowser browser, string html, string url, E
                 throw new Exception("LoadHtml can only be used with the default IResourceHandlerFactory(DefaultResourceHandlerFactory) implementation");
             }
 
-            if (resourceHandler.RegisterHandler(url, ResourceHandler.FromString(html, encoding, true)))
+            if (resourceHandler.RegisterHandler(url, ResourceHandler.FromString(html, encoding, true), false))
```

Comment:
Whilst this change makes sense it is however a subtle change in behavior and could potentially bring some existing users unstuck. There's now a `oneTimeUse` param with a default value as part of `LoadHtml`.